RealmSwift/Tests/ListTests.swift

```diff
@@ -148,7 +148,7 @@ class ListTests: TestCase {
         XCTAssertEqual(str2, array[0])
         XCTAssertEqual(str1, array[1])
 
-        assertThrows(self.array[200])
+        assertThrows(self.array[200], named: nil)
```

Comment:
Why don't we care which exception is thrown in this case? Are there multiple valid exceptions that could be thrown here? I can see some benefit to having all uses of `assertThrows` check the name if that's possible. Otherwise an alternative name for the non-checking case does seem clearer than passing a nil name.